diff -uprN AdminLTE-5.7/scripts/pi-hole/php/footer.php AdminLTE-5.7.cust/scripts/pi-hole/php/footer.php
--- AdminLTE-5.7/scripts/pi-hole/php/footer.php	2021-09-29 22:45:54.000000000 +0200
+++ AdminLTE-5.7.cust/scripts/pi-hole/php/footer.php	2021-09-29 23:00:50.284695509 +0200
@@ -55,9 +55,9 @@
     $ftlUrl = $githubBaseUrl . "/FTL";
     $webUrl = $githubBaseUrl . "/AdminLTE";
 
-    $coreReleasesUrl = $coreUrl . "/releases";
-    $ftlReleasesUrl = $ftlUrl . "/releases";
-    $webReleasesUrl = $webUrl . "/releases";
+    $coreReleasesUrl = "https://aur.archlinux.org/packages/pi-hole-server";
+    $ftlReleasesUrl = "https://aur.archlinux.org/packages/pi-hole-ftl";
+    $webReleasesUrl = "https://aur.archlinux.org/packages/pi-hole-server";
 ?>
     <footer class="main-footer">
         <div class="row row-centered text-center">
@@ -78,18 +78,18 @@
                 <ul class="list-inline">
                     <li>
                         <strong>Pi-hole</strong>
-                        <a href="<?php echo $coreReleasesUrl . "/" . $core_current; ?>" rel="noopener" target="_blank"><?php echo $core_current; ?></a>
-                        <?php if ($core_update) { ?> &middot; <a class="lookatme" href="<?php echo $coreReleasesUrl . "/latest"; ?>" rel="noopener" target="_blank">Update available!</a><?php } ?>
+                        <a href="<?php echo $coreReleasesUrl; ?>" rel="noopener" target="_blank"><?php echo $core_current; ?></a>
+                        <?php if ($core_update) { ?> &middot; <a class="lookatme" href="<?php echo $coreReleasesUrl; ?>" rel="noopener" target="_blank">Update available!</a><?php } ?>
                     </li>
                     <li>
                         <strong>FTL</strong>
-                        <a href="<?php echo $ftlReleasesUrl . "/" . $FTL_current; ?>" rel="noopener" target="_blank"><?php echo $FTL_current; ?></a>
-                        <?php if ($FTL_update) { ?> &middot; <a class="lookatme" href="<?php echo $ftlReleasesUrl . "/latest"; ?>" rel="noopener" target="_blank">Update available!</a><?php } ?>
+                        <a href="<?php echo $ftlReleasesUrl; ?>" rel="noopener" target="_blank"><?php echo $FTL_current; ?></a>
+                        <?php if ($FTL_update) { ?> &middot; <a class="lookatme" href="<?php echo $ftlReleasesUrl; ?>" rel="noopener" target="_blank">Update available!</a><?php } ?>
                     </li>
                     <li>
                         <strong>Web Interface</strong>
-                        <a href="<?php echo $webReleasesUrl . "/" . $web_current; ?>" rel="noopener" target="_blank"><?php echo $web_current; ?></a>
-                        <?php if ($web_update) { ?> &middot; <a class="lookatme" href="<?php echo $webReleasesUrl . "/latest"; ?>" rel="noopener" target="_blank">Update available!</a><?php } ?>
+                        <a href="<?php echo $webReleasesUrl; ?>" rel="noopener" target="_blank"><?php echo $web_current; ?></a>
+                        <?php if ($web_update) { ?> &middot; <a class="lookatme" href="<?php echo $webReleasesUrl; ?>" rel="noopener" target="_blank">Update available!</a><?php } ?>
                     </li>
                 </ul>
                 <?php if($core_update || $web_update || $FTL_update) { ?>
diff -uprN AdminLTE-5.7/scripts/pi-hole/php/func.php AdminLTE-5.7.cust/scripts/pi-hole/php/func.php
--- AdminLTE-5.7/scripts/pi-hole/php/func.php	2021-09-29 22:45:54.000000000 +0200
+++ AdminLTE-5.7.cust/scripts/pi-hole/php/func.php	2021-09-29 23:00:50.286695482 +0200
@@ -143,7 +143,7 @@ function pihole_execute($argument_string
     $escaped = escapeshellcmd($argument_string);
     $output = null;
     $return_status = -1;
-    $command = "sudo pihole " . $escaped;
+    $command = "sudo /usr/bin/pihole " . $escaped;
     exec($command, $output, $return_status);
     if($return_status !== 0)
     {
diff -uprN AdminLTE-5.7/scripts/pi-hole/php/gravity.sh.php AdminLTE-5.7.cust/scripts/pi-hole/php/gravity.sh.php
--- AdminLTE-5.7/scripts/pi-hole/php/gravity.sh.php	2021-09-29 22:45:54.000000000 +0200
+++ AdminLTE-5.7.cust/scripts/pi-hole/php/gravity.sh.php	2021-09-29 23:00:50.294695377 +0200
@@ -30,7 +30,7 @@ function echoEvent($datatext) {
     echo "data: ".implode("\ndata: ", explode("\n",$datatext))."\n\n";
 }
 
-$proc = popen("sudo pihole -g", 'r');
+$proc = popen("sudo /usr/bin/pihole -g", 'r');
 while (!feof($proc)) {
     echoEvent(fread($proc, 4096));
 }
diff -uprN AdminLTE-5.7/scripts/pi-hole/php/header.php AdminLTE-5.7.cust/scripts/pi-hole/php/header.php
--- AdminLTE-5.7/scripts/pi-hole/php/header.php	2021-09-29 22:45:54.000000000 +0200
+++ AdminLTE-5.7.cust/scripts/pi-hole/php/header.php	2021-09-29 23:00:50.250695956 +0200
@@ -587,12 +587,6 @@ if($auth) {
                             <i class="fa fa-fw fa-list-ul"></i> Tail pihole-FTL.log
                         </a>
                     </li>
-                    <!-- Generate debug log -->
-                    <li<?php if($scriptname === "debug.php"){ ?> class="active"<?php } ?>>
-                        <a href="debug.php">
-                            <i class="fa fa-fw fa-ambulance"></i> Generate debug log
-                        </a>
-                    </li>
                     <!-- Network -->
                     <li<?php if($scriptname === "network.php"){ ?> class="active"<?php } ?>>
                         <a href="network.php">
diff -uprN AdminLTE-5.7/scripts/pi-hole/php/queryads.php AdminLTE-5.7.cust/scripts/pi-hole/php/queryads.php
--- AdminLTE-5.7/scripts/pi-hole/php/queryads.php	2021-09-29 22:45:54.000000000 +0200
+++ AdminLTE-5.7.cust/scripts/pi-hole/php/queryads.php	2021-09-29 23:00:50.288695456 +0200
@@ -52,7 +52,7 @@ else
     $exact = "";
 }
 
-$proc = popen("sudo pihole -q -adlist ".$url." ".$exact, 'r');
+$proc = popen("sudo /usr/bin/pihole -q -adlist ".$url." ".$exact, 'r');
 while (!feof($proc)) {
     echoEvent(fread($proc, 4096));
 }
diff -uprN AdminLTE-5.7/scripts/pi-hole/php/savesettings.php AdminLTE-5.7.cust/scripts/pi-hole/php/savesettings.php
--- AdminLTE-5.7/scripts/pi-hole/php/savesettings.php	2021-09-29 22:45:54.000000000 +0200
+++ AdminLTE-5.7.cust/scripts/pi-hole/php/savesettings.php	2021-09-29 23:00:50.291695416 +0200
@@ -510,7 +510,7 @@ function addStaticDHCPLease($mac, $ip, $
 				{
 					global $available_themes;
 					if(array_key_exists($_POST["webtheme"], $available_themes))
-						exec('sudo pihole -a theme '.$_POST["webtheme"]);
+						exec('sudo /usr/bin/pihole -a theme '.$_POST["webtheme"]);
 				}
 				$success .= "The webUI settings have been updated";
 				break;
diff -uprN AdminLTE-5.7/scripts/pi-hole/php/tailLog.php AdminLTE-5.7.cust/scripts/pi-hole/php/tailLog.php
--- AdminLTE-5.7/scripts/pi-hole/php/tailLog.php	2021-09-29 22:45:54.000000000 +0200
+++ AdminLTE-5.7.cust/scripts/pi-hole/php/tailLog.php	2021-09-29 23:00:50.255695891 +0200
@@ -14,11 +14,11 @@ if(!$auth) die("Not authorized");
 // files as large as the pihole.log
 if(isset($_GET["FTL"]))
 {
-	$file = fopen("/var/log/pihole-FTL.log","r");
+	$file = fopen("/run/log/pihole-ftl/pihole-FTL.log","r");
 }
 else
 {
-	$file = fopen("/var/log/pihole.log","r");
+	$file = fopen("/run/log/pihole/pihole.log","r");
 }
 
 if(!$file)
diff -uprN AdminLTE-5.7/scripts/pi-hole/php/update_checker.php AdminLTE-5.7.cust/scripts/pi-hole/php/update_checker.php
--- AdminLTE-5.7/scripts/pi-hole/php/update_checker.php	2021-09-29 22:45:54.000000000 +0200
+++ AdminLTE-5.7.cust/scripts/pi-hole/php/update_checker.php	2021-09-29 23:00:50.266695746 +0200
@@ -7,12 +7,12 @@ $GitHubVersions = "/etc/pihole/GitHubVer
 if(!is_readable($localversions) || !is_readable($localbranches) || !is_readable($GitHubVersions))
 {
 	$core_branch = "master";
-	$core_current = "N/A";
+	$core_current = "5.5";
 	$core_update = false;
 	$web_branch = "master";
-	$web_current = "N/A";
+	$web_current = "5.7";
 	$web_update = false;
-	$FTL_current = "N/A";
+	$FTL_current = exec("pihole-FTL version");
 	$FTL_update = false;
 }
 else
@@ -25,31 +25,31 @@ else
 	// Check if on a dev branch
 	$core_branch = $branches[0];
 	if($core_branch !== "master") {
-	    $core_current = "vDev";
+	    $core_current = "5.5";
 	    $core_commit = $versions[0];
 	}
 	else {
-	    $core_current = explode("-",$versions[0])[0];
+	    $core_current = "5.5";
 	}
 
 	/********** Get Pi-hole web branch / version / commit **********/
 	$web_branch = $branches[1];
 	if($web_branch !== "master") {
-	    $web_current = "vDev";
+	    $web_current = "5.7";
 	    $web_commit = $versions[1];
 	}
 	else {
-	    $web_current = explode("-",$versions[1])[0];
+	    $web_current = "5.7";
 	}
 
 	/********** Get Pi-hole FTL (not a git repository) **********/
 	$FTL_branch = $branches[2];
 	if(substr($versions[2], 0, 4) === "vDev") {
-	    $FTL_current = "vDev";
+	    $FTL_current = exec("pihole-FTL version");
 	    $FTL_commit = $versions[2];
 	}
 	else {
-	    $FTL_current = $versions[2];
+	    $FTL_current = exec("pihole-FTL version");
 	}
 
 	// Get data from GitHub
