--- pi-hole/advanced/Scripts/updatecheck.sh	2023-05-30 20:27:52.000000000 +0200
+++ pi-hole.a/advanced/Scripts/updatecheck.sh	2023-08-21 18:58:07.565225054 +0200
@@ -8,21 +8,20 @@
 # This file is copyright under the latest version of the EUPL.
 # Please see LICENSE file for your rights under this license.
 
+declare -A arch_versions
+arch_versions["/etc/.pihole"]="{{_pkgver}}"
+arch_versions["/var/www/html/admin"]="{{_wwwpkgver}}"
+
 function get_local_branch() {
-    # Return active branch
-    cd "${1}" 2> /dev/null || return 1
-    git rev-parse --abbrev-ref HEAD || return 1
+    echo "master" || return 1
 }
 
 function get_local_version() {
-    # Return active version
-    cd "${1}" 2> /dev/null || return 1
-    git describe --tags --always 2> /dev/null || return 1
+    echo "${arch_versions["${1}"]}" || return 1
 }
 
 function get_local_hash() {
-    cd "${1}" 2> /dev/null || return 1
-    git rev-parse --short=8 HEAD || return 1
+    echo "builtfromreleasetarball" || return 1
 }
 
 function get_remote_version() {
@@ -31,7 +30,7 @@
 
 
 function get_remote_hash(){
-    git ls-remote "https://github.com/pi-hole/${1}" --tags "${2}" | awk '{print substr($0, 0,8);}' || return 1
+    echo "builtfromreleasetarball" || return 1
 }
 
 # Source the setupvars config file
