--- pi-hole/advanced/Scripts/version.sh	2020-12-02 22:37:42.000000000 +0100
+++ pi-hole.a/advanced/Scripts/version.sh	2020-12-06 00:52:13.678272286 +0100
@@ -9,7 +9,7 @@
 # Please see LICENSE file for your rights under this license.
 
 # Variables
-DEFAULT="-1"
+DEFAULT=""
 COREGITDIR="/etc/.pihole/"
 WEBGITDIR="/var/www/html/admin/"
 
@@ -24,19 +24,10 @@
         return 0
     fi
 
-    # Get the tagged version of the local repository
-    local directory="${1}"
-    local version
-
-    cd "${directory}" 2> /dev/null || { echo "${DEFAULT}"; return 1; }
-    version=$(git describe --tags --always || echo "$DEFAULT")
-    if [[ "${version}" =~ ^v ]]; then
-        echo "${version}"
-    elif [[ "${version}" == "${DEFAULT}" ]]; then
-        echo "ERROR"
-        return 1
+    if [[ "$1" == "pi-hole" ]]; then
+        echo "{{_pkgver}}"
     else
-        echo "Untagged"
+        echo "{{_wwwpkgver}}"
     fi
     return 0
 }
